3-UsingAPIs/Week2/homework/ex1-programmerFun/index.js

```diff
@@ -17,27 +17,41 @@
 ------------------------------------------------------------------------------*/
 function requestData(url) {
   // TODO return a promise using `fetch()`
+  return fetch(url)
+  .then((respond)=>{
+    return respond.json()
+  })
+  .catch((error)=>{
+    console.log(error)
+  })
```

Comment:
Good work, we would also like to add HTTP error code handling. Can you add it here?